package.json

```diff
@@ -32,7 +32,9 @@
     "clean:all": "lerna clean --yes && rm -rf node_modules",
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
-    "compile": "lerna run compile",
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:build": "lerna ls | cut -d ' ' -f1 | xargs -I {} sh -c './node_modules/.bin/babel packages/{}/src --out-dir packages/{}/lib --copy-files'",
```

Comment:
could you explain this?

Reply:
This lists out all the terra packages, loops through them and runs babel on the src directory and outputs to the lib directory. Might be helpful to post time comparisons of running `compile` before and after this change.

Reply:
I'll add some screenshots to the issue.

Reply:
I agree it is nice to speed this up. out of curiosity, how does the following compare?
```
"compile": "lerna run compile --parallel"
```
For me, I notice it took 62 ms to run this vs running the original script which took 86 sec on my machine.

Reply:
Running `compile` in master today, these are the times I see on my device:
```
npm run compile  129.51s user 20.61s system 210% cpu 1:11.27 total
```

Running `"compile": "lerna run compile --parallel"`, these are the times I see on my device:
```
npm run compile  161.44s user 26.73s system 419% cpu 44.868 total
```

Running `compile` with the changes in this PR, these are the times I see on my device:
```
npm run compile  52.09s user 6.54s system 121% cpu 48.364 total
```

Reply:
I had looked into the --parallel option previously, but I got about what @bjankord was getting above, immediately following a fresh clone / npm install. The times are actually even higher if it runs as part of the post install step.

Reply:
I'm fine with the code as-is in the PR, it looks to be the fastest/least CPU intensive.